Header header
string name
geometry_msgs/PoseStamped pose
float64 confidence
